Dockerfile

```diff
@@ -9,16 +9,21 @@ LABEL go.version=${GO_VER}
 LABEL ginkgo.version=${GINKGO_VER}
 LABEL container.user=${CONTAINERUSER}
 
-ENV PATH "$PATH:/usr/local/go/bin:/root/go/bin"
+ENV GOPATH="/usr/local/go"
+ENV PATH "$PATH:$GOPATH/bin"
+
 RUN dnf install -y tar gcc make && \
     dnf clean metadata packages && \
     arch=$(arch | sed s/aarch64/arm64/ | sed s/x86_64/amd64/) && \
     curl -Ls https://go.dev/dl/${GO_VER}.linux-${arch}.tar.gz |tar -C /usr/local -xzf -  && \
-    useradd -U -u 1000 -m -d /home/${CONTAINERUSER} -s /usr/bin/bash ${CONTAINERUSER}
+    go install github.com/onsi/ginkgo/v2/${GINKGO_VER} && \
+    useradd -U -u 1000 -m -d /home/${CONTAINERUSER} -s /usr/bin/bash ${CONTAINERUSER} && \
+    mkdir /eco-gotests && chown ${CONTAINERUSER}:${CONTAINERUSER} /eco-gotests && \
+    echo 'RUN done'
```

Comment:
This isn't necessary

Reply:
What is not necessary?

Reply:
```suggestion
```

Reply:
Why do u think I put it?    It's a way to add commands to the RUN, and always use the && \, and worry about remove it from last line..

Reply:
Please merge this.  the ocp-eco-gotests.groovy is pending on this.